src/main/java/org/jabref/logic/remote/server/RemoteListenerServerLifecycle.java

```diff
@@ -43,7 +43,9 @@ public void open(MessageHandler messageHandler, int port, PreferencesService pre
         try {
             result = new RemoteListenerServerThread(messageHandler, port, preferencesService);
         } catch (BindException e) {
-            LOGGER.warn("Port is blocked", e);
+            LOGGER.warn("There was an error opening the configured network port {}. Please ensure there isn't another" +
+                    " application already using that port. If you're sure an existing JabRef instance is running and" +
+                    " using this port, and you see this message, please file a bug.", port);
```

Comment:
Thanks for your effort in improving JabRef user experience. Changes look good, except one small nitpick:
The message seems to be a bit too detailed. We could put the third sentence below almost every logger message...
Be aware that the Logger already provides extensive information about the class where the warning has risen etc.
I believe the first two sentences are sufficient about the port in use.

Reply:
I appreciate the feedback! I wanted the message to try to cover both potential cases:

1. Some other application is already using that port, so we can't bind it
2. *JabRef* is already using that port, and the ping timed out even after increasing to 1s

If we remove the third sentence, we lose point number 2. I encountered this issue and the only clue I got was that JabRef couldn't bind the port, nothing about the ping failing. Without the third sentence, a future user who hits the timeout issue will face the same lack of clues that I did. Do you have a suggestion for how we could phrase the message to cover both points? Or do you think that, post change, this scenario is unlikely enough to be encountered that it's better to shorten the message at the expense of that information? I'm happy to remove it if you think that's the better course.

Reply:
Sentences are:
1. There was an error opening the configured network port {}.
2. Please ensure there isn't another application already using that port.
3. If you're sure an existing JabRef instance is running and using this port, and you see this message, please file a bug.

The meaning of the first part of sentence three is implicitly covered by the second. I expect the interested user who actually reads the log files to be clever enough to reach out to our support resources. ;-)

Please just remove sentence three...

Reply:
i think we should put that in the documentation for troubleshooting

Reply:
Done :)

Reply:
Where in the documentation can I find it? I checked https://github.com/JabRef/user-documentation/pulls, however I don't find an open (or closed) PR.